src/app/maker/maker-history-shifts/maker-history-shifts.component.html

```diff
@@ -0,0 +1,16 @@
+<p>maker-history-shifts works!</p>
+<ion-content makerShifts-history>
+  <ion-title><h3>Shifts History</h3></ion-title>
+  <ul>
+    <!-- v-- hrefs below - CHANGE to ionic or ang routerLink --v -->
+    <li><ion-card><a href><h4>Anniversary Brunch</h4><p></p>
```

Comment:
these hrefs should become function calls eventually, but they're fine dummy entries at the moment